packages/app/src/cli/models/extensions/specifications/app_access.ts

```diff
@@ -0,0 +1,16 @@
+import {AppSchema} from '../../app/app.js'
+import {APP_ACCESS_IDENTIFIER} from '../app-config.js'
+import {BaseSchema} from '../schemas.js'
+import {createExtensionSpecification} from '../specification.js'
+
+// We are calling `strip()` to remove the `strict()` setting in the base AppSchema
+const AppAccessSchema = BaseSchema.merge(AppSchema.pick({access: true})).strip()
```

Comment:
um, but do you actually need anything else from `BaseSchema` or `AppSchema`?
`BaseSchema` is the base for extension tomls, but it shouldn't apply for an app config, maybe you just need
`AppSchema.pick({access: true}).strip()`?

But if you only care about `access`, then let's extract that schema:

```ts
const AccessSchema = zod
      .object({
        direct_api_offline_access: zod.boolean().optional(),
      })
      .optional(),
```

and you can use that schema both in `AppSchema` and here. Does that make sense?

Reply:
I see, that makes sense, well at least almost everything in there is optional anyway